### ISSUE TITLE
gsk4-sys test failures with gtk4 >= 4.9.4

### ISSUE THREAD

**Issue by decathorpe** [gtk-rs/gtk4-rs#1316]
I'm responsible for packaging gtk-rs, gtk4-rs, and gstreamer-rs for Fedora Linux. Our CI shows that the update from gtk4 4.9.3 to 4.9.4 causes the test suite in the gsk4-sys crate (we currently have packaged v0.5.5 and v0.4.8) to fail:

```
test cross_validate_constants_with_c ... FAILED
failures:
---- cross_validate_constants_with_c stdout ----
Constant value mismatch for (gint) GSK_BLEND_NODE
Rust: "20"
C:    "21"
Constant value mismatch for (gint) GSK_BLUR_NODE
Rust: "23"
C:    "25"
Constant value mismatch for (gint) GSK_CLIP_NODE
Rust: "17"
C:    "18"
Constant value mismatch for (gint) GSK_COLOR_MATRIX_NODE
Rust: "15"
C:    "16"
Constant value mismatch for (gint) GSK_CROSS_FADE_NODE
Rust: "21"
C:    "23"
Constant value mismatch for (gint) GSK_DEBUG_NODE
Rust: "24"
C:    "26"
Constant value mismatch for (gint) GSK_GL_SHADER_NODE
Rust: "25"
C:    "27"
Constant value mismatch for (gint) GSK_INSET_SHADOW_NODE
Rust: "11"
C:    "12"
Constant value mismatch for (gint) GSK_OPACITY_NODE
Rust: "14"
C:    "15"
Constant value mismatch for (gint) GSK_OUTSET_SHADOW_NODE
Rust: "12"
C:    "13"
Constant value mismatch for (gint) GSK_REPEAT_NODE
Rust: "16"
C:    "17"
Constant value mismatch for (gint) GSK_ROUNDED_CLIP_NODE
Rust: "18"
C:    "19"
Constant value mismatch for (gint) GSK_SHADOW_NODE
Rust: "19"
C:    "20"
Constant value mismatch for (gint) GSK_TEXT_NODE
Rust: "22"
C:    "24"
Constant value mismatch for (gint) GSK_TRANSFORM_NODE
Rust: "13"
C:    "14"
thread 'cross_validate_constants_with_c' panicked at 'FAILED: 52 passed; 15 failed', tests/abi.rs:107:13
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
failures:
    cross_validate_constants_with_c
test result: FAILED. 1 passed; 1 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.87s
```

This looks worrying to me. Does this mean that applications that use these constants via the Rust bindings are broken with gtk4 >= 4.9.4?

**Comment by bilelmoussaoui** [comment]
Not a bindings issue, it was fixed in https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/5519

**Comment by decathorpe** [comment]
Thanks for the pointer! Looks like this fix happened after 4.9.4 and there's no new release yet :(
